pywbemtools/pywbemcli/_cmd_server.py

```diff
@@ -412,3 +437,138 @@ def cmd_server_remove_mof(context, options):
+            if len(schema_elements) < 2:
+                no_schema.append(klass.classname)
+            if schema not in schema_max_ver:
+                schema_max_ver[schema] = [0, 0, 0]
+
+            # If detail, get the experimental qualifier for this schema
+            if detail:
+                if schema not in schema_experimental:
+                    schema_experimental[schema] = False
+                if this_class_experimental:
+                    schema_experimental[schema] = True
+                elif ns_experimental:
+                    if schema_experimental[schema] is False:
+                        if is_experimental_class(klass):
+                            schema_experimental[schema] = True
+
+            # Get the version qualifier for this class
+            if 'Version' in klass.qualifiers:
+                version = klass.qualifiers['Version'].value
+                version = parse_version_value(version, klass.classname)
+
+                if version > ns_max_version:
+                    ns_max_version = version
+
+                if detail:
+                    if schema not in schema_max_ver or \
+                            version > schema_max_ver[schema]:
+                        schema_max_ver[schema] = version
+
+        # Build the table formatted output
+        prev_namespace = None
+        ns_version_str = version_str(ns_max_version) if classes_count else ""
+
+        if detail:
+            # Display with a line for each namespace and one for each
+            # schema in the namespace
+            for schema in sorted(schemas.keys()):
+                schema_max_ver_str = version_str(schema_max_ver[schema])
+                # Append a row for each new namespace
+                if ns != prev_namespace:
+                    prev_namespace = ns
+                    rows.append([ns,
+                                 "(namespace-info)",
+                                 classes_count,
+                                 ns_version_str,
+                                 experimental_display(ns_experimental)])
+                # Append the row for each schema in the namespace
+                rows.append(["",
+                             schema,
+                             schemas[schema],
+                             schema_max_ver_str,
+                             experimental_display(schema_experimental[schema])])
+        else:  # display non-detail report
+            # Display one line for each namespace with list of schemas in the
+            # namespace
+            schemas_str = ", ".join(sorted(list(six.iterkeys(schemas))))
+            folded_schemas = fold_strings(schemas_str, 45,
+                                          fold_list_items=False)
+            rows.append([ns,
+                         folded_schemas,
+                         classes_count,
+                         ns_version_str,
+                         experimental_display(ns_experimental)
+                         ])
```

Comment:
It is confusing to show a schema version in the non-detailed output, since you are applying the max value on possibly very different version number ranges. I suggest to omit the version column in the non-detailed report.

Reply:
That is the one data item that this was st

Reply:
So if a vendor schema version 5.3, and the DMTF CIM schema version is 2.50, then this column in the non-detail case will show 5.3. Does that make sense? I think it does not. A list if different schemas does not have a single version.

Reply:
Agree, discussion.  Sadly one thing we know is that generally it is the DMTF schema that is the basic driver of versions.   Without the versions, what you see in the non-detail output is for each namespace a list of the schema, the number of classes, and whether there was experimental anywhere in the namespace. No clue about the dmtf schema.  However, you did point out one major error, I am using the max version number as the max dmtf schema version.  I have to isolate that search for max version to classes that are in one of the DMTF schema (CIM, PRS, ???). If someone put a schema version 99.99.00 on a class in their schema I would record that as max in the non-detail view.  That is just a bug.

Reply:
So you would silently show only the DMTF schema version, ignoring any other schema's versions, without saying so?? In Python there is the principle pof least surprise. Don't think that would be met here...